package.json

```
@@ -14,15 +14,16 @@
   ],
   "type": "module",
   "scripts": {
-    "build": "rm -rf ./dist && tsc -b src",
+    "build": "rimraf ./dist && cross-env tsc -b src",
     "start": "parcel playground/index.html --open --dist-dir ./public",
-    "build:playground": "rm -rf ./public && parcel build playground/index.html --no-scope-hoist --dist-dir ./public --public-url /",
+    "build:playground": "rimraf ./public && parcel build playground/index.html --no-scope-hoist --dist-dir ./public --public-url /",
     "test:code": "eslint --max-warnings=0 --ext .js,.ts,.tsx ."
   },
   "dependencies": {
     "@excalidraw/markdown-to-text": "0.1.2",
     "mermaid": "10.2.3",
-    "nanoid": "4.0.2"
+    "nanoid": "4.0.2",
+    "cross-env": "^7.0.3"
```

```suggestion
    "cross-env": "7.0.3"
```
lets lock the exact version so everyone has exact same build when developing